content/articles/add_docs.md

```diff
@@ -0,0 +1,46 @@
+Title: How to add docs to this site
+Date: 11-29-2018
+Tags: how-to, docs
+Slug: how-to-add-docs
+Authors: Omar Quimbaya
+Summary: A short tutorial of how to add your docs to this site and share your knowledge!
+
+# How to add documentation to the SA Devs website
+
+First of all, why would you want to do this? We are a group of developers who are active in our Slack group, but one of the main problems is that we have a lot of knowledge that we are not writing down in a more permanent place. When we share articles or advice on a technical topic, those messages will get lost with all of the other messages that people are sending as we are not paying for our Slack subscription. Therefore, in order to share our knowledge with people and show other cities that the developers in San Antonio are knowledgeable, collaborative, and willing to share their experience with the rest of the world, we are giving developers here an opportunity to share articles on their thoughts, experiences, and knowledge of how to use the technologies we use every day.
+
+This is a good opportunity for you to contribute something to the community without that much effort! In order to do so, here is how you do it.
+
+## Fork the repo!
+
+Our site is a static site hosted somewhere out there on the Internet. Super mysterious. You can find the repository of our site here: [https://github.com/SADevs/sadevs.github.io](https://github.com/SADevs/sadevs.github.io). Fork this repository and clone it to your local machine. The site is powered by Pelican, which is a static site generator written in Python. If you have Python on your machine and you want to see what your article looks like on your local machine, install Pelican by typing in your terminal `pip install pelican Markdown`. This will get the necessary packages to use this static site generator.
+
```

Comment:
please specify using pipenv instead of pip install.

```
pip install pipenv
pipenv install
pipenv run pelican content -vvv -o output -s pelicanconf.py
```

and add using `python -m http.server` to host the static content so it can be checked.